snark-verifier-sdk/examples/StandardPlonkVerifierExample.sol

```diff
@@ -0,0 +1,2115 @@
+
```

Comment:
I feel we could remove this file considering it's fairly easy to reproduce.

Reply:
OK, done!